calyx-py/calyx/queue_call.py

```diff
@@ -0,0 +1,135 @@
+# pylint: disable=import-error
+import calyx.builder as cb
+import calyx.builder_util as util
+
+
+def insert_raise_err_if_i_eq_15(prog):
+    """Inserts a the component `raise_err_if_i_eq_15` into the program.
+
+    It has:
+    - one input, `i`.
+    - one ref register, `err`.
+
+    If `i` equals 15, it raises the `err` flag.
```

Comment:
`15` seems like a magic constant here. If it is, can we define it as the file level using `MAGIC_CONST = 15` (with an actually useful name) after the imports?

Reply:
Thanks, done!